src/FM.LiveSwitch.Mux.Standard/SessionTracker.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Newtonsoft.Json;
+
+namespace FM.LiveSwitch.Mux
+{
+    public class SessionTracker
```

Comment:
If this class isn't serialized anywhere, we don't need the Newtonsoft.Json dependency.

Reply:
It is used for serialization of SessionTracker to store file size tracking of the orphan session candidates in temp file between muxer runs.